pkg/healthcheck/healthcheck.go

```diff
@@ -110,6 +110,10 @@ const (
 	// from LinkerdVersionChecks, so those checks must be added first.
 	LinkerdDataPlaneChecks CategoryID = "linkerd-data-plane"
 
+	// LinkerdHaChecks adds checks to validate that the HA configuration
+	// is correct. These checks are no ops if linkerd is not in HA mode
+	LinkerdHaChecks CategoryID = "linkerd-ha-checks"
```

Comment:
Nit: Can we uppercase the `Ha` in the variable name? See Go convention [here](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).